packages/replayio/src/utils/browser/launchBrowser.ts

```diff
@@ -38,7 +41,31 @@ export async function launchBrowser(
     processOptions
   );
 
-  console.log(`Recording ${dim("(quit browser to continue)")}`);
+  // Wait until the user quits the browser process OR
+  // until the user presses a key to continue (in which case, we will kill the process)
+  const abortControllerForPrompt = new AbortController();
+  const browserClosedDeferred = createDeferred<void>();
 
-  await spawnProcess(runtimeExecutablePath, args, processOptions);
+  const { data: childProcess } = spawnProcess(runtimeExecutablePath, args, processOptions, {
+    onError: () => {
+      abortControllerForPrompt.abort();
+      browserClosedDeferred.resolveIfPending();
```

Comment:
q: shouldn't this be a rejection? I have no idea when our browser can error but I'd assume that usually that would mean some fatal error and that we won't get useful recordings out of this. It would also be good to at least get some insight that this has happened - do we keep this info anywhere? like in the recording log or somewhere?